src/core/ui/components/drop-in/drop-down.tsx

```diff
@@ -0,0 +1,414 @@
+        '...' +
+        hash.substring(hash.length - 4, hash.length);
+      return [hash, hashDisplay];
+    }
+    return [];
+  }, [user.wallet]);
+
+  if (!initialized) return <ScaleLoader height={35} width={4} />;
+  return (
+    <DropDownDiv>
+      <div
+        style={{
+          cursor: 'pointer',
+          display: 'flex',
+          alignItems: 'center',
+          fontWeight: 500,
+          fontSize: '16px',
+          lineHeight: '20px',
+          border: '1px solid #E6E8EB',
+          borderRadius: '22px',
+        }}
+        onClick={() => {
+          if (user.loggedIn) {
+            setIsOpen((b) => !b);
+          } else {
+            loginNoRedirectNoPopup();
+          }
+        }}
+      >
+        <div
+          style={{
+            margin: user.loggedIn ? 3 : undefined,
+            display: 'flex',
+          }}
+        >
+          {profilePicturePlaceholder}
+        </div>
+        {user.loggedIn && (
+          <>
+            <div
+              style={{
+                marginLeft: 10,
+              }}
+            >
+              {hashDisplay}
+            </div>
+            <div
+              style={{
+                marginLeft: 14,
+                marginRight: 13,
+                transform: isOpen ? 'rotate(180deg)' : undefined,
+              }}
+            >
+              {chevronDown}
+            </div>
+          </>
+        )}
+      </div>
+      {isOpen && (
+        <Content>
+          <Section style={{ borderTop: 'none' }}>
+            {/*<Name>{user.name}</Name>*/}
+            {hashDisplay ? (
+              <Row
+                style={{
+                  ...primaryIdStyle,
+                }}
+              >
+                {hashDisplay}
+                <Icon
+                  style={{ marginLeft: 8, cursor: 'pointer' }}
+                  onClick={() => {
+                    navigator.clipboard.writeText(hash);
+                  }}
+                >
+                  {copyIcon}
+                </Icon>
+              </Row>
+            ) : (
+              <Row
+                style={{
+                  paddingTop: 8,
+                  color: '#2F5FFC',
+                }}
+                onClick={() => {
+                  connect(false);
+                }}
+              >
+                <Icon>{plusIcon}</Icon>
+                Connect web3 wallet
+              </Row>
+            )}
+          </Section>
+          {/*<Section>*/}
+          {/*  <Row>*/}
+          {/*    <Icon>{twitterIcon}</Icon>*/}
+          {/*    @0xhaileym*/}
+          {/*  </Row>*/}
+          {/*  <Row*/}
+          {/*    style={{*/}
+          {/*      paddingTop: 8,*/}
+          {/*    }}*/}
+          {/*  >*/}
+          {/*    <Icon>{googleIcon}</Icon>*/}
+          {/*    haileymiller298@gmail.com*/}
+          {/*  </Row>*/}
+          {/*</Section>*/}
+          <Section>
+            <Row
+              onClick={() => {
+                goToAccountPage();
```

Comment:
This is tricky but shouldn't use that external hook. Perhaps this should jsut console log for now or comment it out.

Reply:
updated